src/templates/cromwell/cromwell-server.template.yaml

```diff
@@ -131,6 +139,7 @@ Resources:
                 - "batch:DescribeJobDefinitions"
                 - "batch:ListJobs"
                 - "batch:DescribeComputeEnvironments"
+                - "ecs:DescribeContainerInstances"
```

Comment:
It's really for diagnostic reasons. It allows the server to output information about the ECS cluster, container and EC2 that are being used to run a batch job. This was useful for tracking down the issues with files not downloading properly in the launch template leading to workers that would fail all jobs they hosted.